pkg/network/v1.go

```diff
@@ -0,0 +1,19 @@
+package network
+
+import (
+	"github.com/vishvananda/netlink"
+	"strings"
+)
+
+// NodeIDv1 returns the node ID as it was calculated in 0-OS v1
+func NodeIDv1() (string, error) {
+	link, err := netlink.LinkByName("zos")
```

Comment:
We never created a zos bridge on v1. and since this is a linux bridge. You have no idea what mac it will get (I don't believe it's even persisted if u deleted and re-added the bridge (on boot for example) . But I think, V1 used the physical network card. (I think this was in the templates)

Reply:
indeex very good, thx